src/app/reviewer/[memberSlug]/study/[studyIdentifier]/review/study-review-buttons.tsx

```diff
@@ -41,18 +43,22 @@ export const StudyReviewButtons: FC<{ study: SelectedStudy; memberSlug: string }
 
     if (study.status === 'APPROVED' && study.approvedAt) {
         return (
-            <Group c="#12B886" gap="0">
-                <CheckCircle weight="fill" />
-                <Text>Approved on {dayjs(study.approvedAt).format('MMM DD, YYYY')}</Text>
+            <Group gap="0.5rem">
+                <CheckCircle weight="fill" size={24} color={theme.colors.green[9]} />
```

Comment:
instead of importing the theme and using it this way you can use the string versions of the color:
```suggestion
                <CheckCircle weight="fill" size={24} color="green.9" />
```

Reply:
I imported `useMantineTheme` because the string versions only work with Mantine components. The `XCircle` is a phosphor component